src/palindrom.js

```diff
@@ -823,6 +765,10 @@ var Palindrom = (function () {
 
   /* backward compatibility */
   global.Puppet = Palindrom;
+  
+  /* Since we have jsonpatch bundled,
+  let's expose it in case anyone needs it */
+  global.jsonpatch = jsonpatch;
```

Comment:
Personally, I don't mind keeping jsonpatch public, as someone may would like to use it for it's own purposes, like any other package/custom element we load from `/sys/`.

>It is quite unexpected to get dependencies exposed in the global namespace. 

Most of our elements do so.